components/Footer/Footer.js

```diff
@@ -1,34 +1,33 @@
-import Image from "next/image";
+import Link from "next/link";
 import styled from "styled-components";
+import BookmarkIcon from "../Icons/Bookmark";
+import ScheduleIcon from "../Icons/Schedule";
 
-export default function Footer() {
+function Footer() {
   return (
     <StyledFooterList>
       <StyledIconContainer>
-        <StyledButton>
-          <Image
-            width={40}
-            height={40}
-            src={"/assets/icons8-calendar-24.png"}
-          />
-        </StyledButton>
-        <p>Schedule</p>
+        <Link href={"/"}>
+          <StyledButton>
```

Comment:
if you use a Link, you dont need an additional button here. Since it is a link that links to another page - Link is enougth.